src/At.vue

```diff
@@ -81,12 +85,13 @@ export default {
       bindsValue: this.value != null,
       customsEmbedded: false,
       hasComposition: false,
-      atwho: null
+      atwho: null,
+      type: ""
     }
   },
   computed: {
     atItems () {
-      return this.at ? [this.at] : this.ats
+      return this.ats.map(at => at.symbol)
```

Comment:
Obviously, changes like this is something breaking against all the versions before this, and will influenced all the users who use this library.